pkg/agent/openflow/multicast.go

```diff
@@ -176,14 +176,16 @@ func (f *featureMulticast) multicastPodMetricFlows(podIP net.IP, podOFPort uint3
 }
 
 func (f *featureMulticast) replayGroups() {
+	var adds []binding.OFEntry
 	f.groupCache.Range(func(id, value interface{}) bool {
 		group := value.(binding.Group)
 		group.Reset()
-		if err := group.Add(); err != nil {
-			klog.ErrorS(err, "Error when replaying cached group", "group", id)
-		}
+		adds = append(adds, group)
 		return true
 	})
+	if err := f.bridge.AddOFEntriesInBundle(adds, nil, nil); err != nil {
```

Comment:
ditto

Reply:
Since we add group in bundle, we cannot use the following method to delete a group since `isInstalled` is not set for the group installed in bundle.
```go
func (g *ofGroup) Delete() error {
	return g.ofctrl.Delete()
}
```

In ofnet:
```go
func (self *Group) Delete() error {
	if self.isInstalled {
		groupMod := openflow15.NewGroupMod()
		groupMod.GroupId = self.ID
		groupMod.Command = openflow15.OFPGC_DELETE
		if err := self.Switch.Send(groupMod); err != nil {
			return err
		}
		// Mark it as unInstalled
		self.isInstalled = false
	}

	// Delete group from switch cache
	return self.Switch.DeleteGroup(self.ID)
}
```